---
title: "Spage2vec: Unsupervised representation of localized spatial gene expression signatures"
feature_image: "/assets/gallery_images/spage2vec.png"
link: "https://tissuumaps.research.it.uu.se/spage2vec/index.html"
---

Spage2vec is an unsupervised segmentation free approach for decrypting the spatial transcriptomic heterogeneity of complex tissues at subcellular resolution. Spage2vec represents the spatial transcriptomic landscape of tissue samples as a graph and leverage powerful machine learning graph representation technique to create a lower dimensional representation of local spatial gene expression. Here we visualize spage2vec localized gene expression signatures of different spatial transcriptomic datasets. \
We thank Mats Nilsson, Sten Linnarsson and Xiaowei Zhuang for making their datasets publicly available.

**TissUUmaps interactive viewer 1**: \
<a href='https://tissuumaps.dckube.scilifelab.se/demo/Spage2vec_ISS_Qian_et_al.tmap' target="_blank" class="button">In situ sequencing mouse brain hippocampal area CA1</a> \
<small>Qian, X., Harris, K. D., Hauling, T., Nicoloutsopoulos, D., Mu√±oz-Manchado, A. B., Skene, N., ... & Nilsson, M. (2020). Probabilistic cell typing enables fine mapping of closely related cell types in situ. [Nature methods](https://doi.org/10.1038/s41592-019-0631-4){:target="_blank"}, 17(1), 101-106.</small>

**TissUUmaps interactive viewer 2**: \
<a href='https://tissuumaps.dckube.scilifelab.se/demo/Spage2vec_osmFISH_Codeluppi_et_al.tmap' target="_blank" class="button">osmFISH mouse brain somatosensory cortex</a> \
<small>Codeluppi, S., Borm, L. E., Zeisel, A., La Manno, G., van Lunteren, J. A., Svensson, C. I., & Linnarsson, S. (2018). Spatial organization of the somatosensory cortex revealed by osmFISH. [Nature methods](https://doi.org/10.1038/s41592-018-0175-z){:target="_blank"}, 15(11), 932-935.</small>

**TissUUmaps interactive viewer 3**: \
<a href='https://tissuumaps.dckube.scilifelab.se/demo/Spage2vec_MERFISH_Moffitt_et_al.tmap' target="_blank" class="button">MERFISH mouse brain hypothalamic preoptic area</a> \
<small>Moffitt, J. R., Bambah-Mukku, D., Eichhorn, S. W., Vaughn, E., Shekhar, K., Perez, J. D., ... & Zhuang, X. (2018). Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region. [Science](https://doi.org/10.1126/science.aau5324){:target="_blank"}, 362(6416), eaau5324.</small>

**TissUUmaps interactive viewer 4**: \
<a href='https://tissuumaps.dckube.scilifelab.se/demo/Spage2vec_MERFISH_Chen_et_al_2015.tmap' target="_blank" class="button">MERFISH human fibroblast cells (IMR90)</a> \
<small>Chen, K. H., Boettiger, A. N., Moffitt, J. R., Wang, S., & Zhuang, X. (2015). Spatially resolved, highly multiplexed RNA profiling in single cells. [Science](https://doi.org/10.1126/science.aaa6090){:target="_blank"}, 348(6233), aaa6090.</small>

Data and code availability: Spatial gene expression data are available in Zenodo database at <https://doi.org/10.5281/zenodo.3897401>{:target="_blank"}.
Source code for reproducing analysis results and figures is available in Zenodo database at <http://www.doi.org/10.5281/zenodo.4030404>{:target="_blank"}.