---
title: "Spage2vec: Unsupervised representation of localized spatial gene expression signatures"
feature_image: "/assets/gallery_images/spage2vec.png"
link: "https://tissuumaps.research.it.uu.se/spage2vec/index.html"
---

Spage2vec is an unsupervised segmentation free approach for decrypting the spatial transcriptomic heterogeneity of complex tissues at subcellular resolution. Spage2vec represents the spatial transcriptomic landscape of tissue samples as a graph and leverage powerful machine learning graph representation technique to create a lower dimensional representation of local spatial gene expression. Here we visualize spage2vec localized gene expression signatures of different spatial transcriptomic datasets. \
We thank Mats Nilsson, Sten Linnarsson and Xiaowei Zhuang for making their datasets publicly available.

**Dataset 1**: In situ sequencing mouse brain hippocampal area CA1: \
<a href="https://tissuumaps.research.it.uu.se/demo/ISS_Qian_et_al.html">https://tissuumaps.research.it.uu.se/demo/ISS_Qian_et_al.html</a> \
Qian, X., Harris, K. D., Hauling, T., Nicoloutsopoulos, D., Mu√±oz-Manchado, A. B., Skene, N., ... & Nilsson, M. (2020). Probabilistic cell typing enables fine mapping of closely related cell types in situ. <a href="https://doi.org/10.1038/s41592-019-0631-4"> Nature methods</a>, 17(1), 101-106. 

**Dataset 2**: osmFISH mouse brain somatosensory cortex: \
<a href="https://tissuumaps.research.it.uu.se/demo/osmFISH_Codeluppi_et_al.html">https://tissuumaps.research.it.uu.se/demo/osmFISH_Codeluppi_et_al.html</a> \
Codeluppi, S., Borm, L. E., Zeisel, A., La Manno, G., van Lunteren, J. A., Svensson, C. I., & Linnarsson, S. (2018). Spatial organization of the somatosensory cortex revealed by osmFISH. <a href=" https://doi.org/10.1038/s41592-018-0175-z"> Nature methods</a>, 15(11), 932-935. 

**Dataset 3**: MERFISH mouse brain hypothalamic preoptic area: \
<a href="https://tissuumaps.research.it.uu.se/demo/MERFISH_Moffitt_et_al.html">https://tissuumaps.research.it.uu.se/demo/MERFISH_Moffitt_et_al.html</a> \
Moffitt, J. R., Bambah-Mukku, D., Eichhorn, S. W., Vaughn, E., Shekhar, K., Perez, J. D., ... & Zhuang, X. (2018). Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region. <a href="https://doi.org/10.1126/science.aau5324">Science</a>, 362(6416), eaau5324. 

**Dataset 4**: MERFISH human fibroblast cells (IMR90): \
<a href="https://tissuumaps.research.it.uu.se/demo/MERFISH_Chen_et_al_2015.html">https://tissuumaps.research.it.uu.se/demo/MERFISH_Chen_et_al_2015.html</a> \
Chen, K. H., Boettiger, A. N., Moffitt, J. R., Wang, S., & Zhuang, X. (2015). Spatially resolved, highly multiplexed RNA profiling in single cells. <a href="https://doi.org/10.1126/science.aaa6090">Science</a>, 348(6233), aaa6090.

Data and code availability: Spatial gene expression data are available in Zenodo database at <a href="https://doi.org/10.5281/zenodo.3897401">https://doi.org/10.5281/zenodo.3897401</a>. Source code for reproducing analysis results and figures is available in Zenodo database at <a href="http://www.doi.org/10.5281/zenodo.4030404">http://www.doi.org/10.5281/zenodo.4030404</a>.